app/renderer/js/utils/link-util.ts

```diff
@@ -19,24 +19,24 @@ export async function openBrowser(url: URL): Promise<void> {
 		);
 		const file = path.join(dir, 'redirect.html');
 		fs.writeFileSync(file, `\
-<!DOCTYPE html>
-<html>
-    <head>
-        <meta charset="UTF-8" />
-        <meta http-equiv="Refresh" content="0; url=${escape(url.href)}" />
-        <title>Redirecting</title>
-        <style>
-            html {
-                font-family: menu, "Helvetica Neue", sans-serif;
-            }
-        </style>
-    </head>
-    <body>
-        <p>Opening <a href="${escape(url.href)}">${escape(url.href)}</a>…</p>
-    </body>
-</html>
-`);
-		shell.openItem(file);
+		<!DOCTYPE html>
+		<html>
+			<head>
+				<meta charset="UTF-8" />
+				<meta http-equiv="Refresh" content="0; url=${escape(url.href)}" />
+				<title>Redirecting</title>
+				<style>
+					html {
+						font-family: menu, "Helvetica Neue", sans-serif;
+					}
+				</style>
+			</head>
+			<body>
+				<p>Opening <a href="${escape(url.href)}">${escape(url.href)}</a>…</p>
+			</body>
+		</html>
+		`);
+		await shell.openExternal(file); // Check
```

Comment:
The [documentation](https://www.electronjs.org/docs/breaking-changes) says that `shell.openItem` was replaced with `shell.openPath`, which takes a path, not with `shell.openExternal`, which takes a URL.